tests/saw/verify_HMAC_bad_magic_mod.saw

```diff
@@ -0,0 +1,13 @@
+
+include "HMAC/spec/verify_HMAC.saw";
+
+let sha256_bad_magic_mod_cfg =
+  { name            = "SHA256_bad_magic_mod"
+  , hmac_alg        = {{ S2N_HMAC_SHA256 }}
+  , digest_size     = 32
+  , block_size      = 64
+  , hash_block_size = 64
+  , highest_32_bit  = 4294949761
```

Comment:
can you put a comment as to what we're doing here?

Reply:
Added! 67efc5f30c7db395423a67532e5f9ca8f62660d7